packages/@lwc/ssr-compiler/src/__tests__/estemplate.spec.ts

```diff
@@ -57,7 +57,7 @@ describe.each(
                 `;
             const doReplacement = () => tmpl(b.literal('I am not an identifier') as any);
             expect(doReplacement).toThrow(
-                "Validation failed for template. Expected node of type 'Identifier' but received node of type 'Literal'."
+                'Validation failed for templated node of type Identifier'
```

Comment:
```suggestion
                'Validation failed for templated node of type Literal'
```

Reply:
This is just a pure revert, `Identifier` is what was there before.